ui/shared/components/panel/variants/VariantUtils.jsx

```diff
@@ -60,3 +60,29 @@ export const ProteinSequence = React.memo(({ hgvs }) => <Sequence color="black"
 ProteinSequence.propTypes = {
   hgvs: PropTypes.string.isRequired,
 }
+
+const variantIntervalOverlap = (variant, padding, noGenomeVersion) => (interval) => {
+  const { pos, end, genomeVersion, liftedOverPos } = variant
+  const variantPos = (genomeVersion === interval.genomeVersion) || noGenomeVersion ? pos : liftedOverPos
+  if (!variantPos) {
+    return false
+  }
+  if ((variantPos >= interval.start - padding) && (variantPos <= interval.end + padding)) {
+    return true
+  }
+  if (end && !variant.endChrom) {
+    const variantPosEnd = variantPos + (end - pos)
+    return (variantPosEnd >= interval.start - padding) && (variantPosEnd <= interval.end + padding)
+  }
+  return false
+}
+
+export const getOverlappedIntervals = (
+  variant, intervals, familiesByGuid = null, padding = 0, noGenomeVersion = false,
+) => {
+  const { familyGuids = [] } = variant
+  const familyIntervals = familyGuids.reduce((acc, fGuid) => (
+    [...acc, ...(intervals[familiesByGuid ? familiesByGuid[fGuid].projectId : fGuid] || [])]), [])
```

Comment:
It feels a little magical that if `familiesByGuid` is defined it should use `projectId` but otherwise not. I think it would be better to be more explicit and instead of `familiesByGuid` as the parameter, have `getIntervalGroupId` as a a parameter and then here it could be `intervals[getIntervalGroupId(fGuid)]`